src/components/EscDshotDirection/Styles.css

```diff
@@ -0,0 +1,158 @@
+.escDshotDirection-Component {
+    display: flex;
+    height: 100%;
+    flex-flow: column;
+}
+
+.escDshotDirection-ComponentHeader {
+    padding-bottom: 12px;
+}
+
+#escDshotDirectionDialog-MainContent {
+    display: flex;
+    height: 100%;
+    flex-flow: column;
+}
+
+#escDshotDirectionDialog-Warning {
+    display: flex;
+    height: 100%;
+    flex-flow: column;
+}
+
+.escDshotDirectionToggleParentContainer {
+    display: flex;
+    margin-bottom: 1.5em;
+    margin-top: 1.5em;
+}
+
+.escDshotDirectionToggleNarrow {
+    margin-right: 12px;
+    display: flex;
+    align-items: center;
+}
+
+.escDshotDirectionDialog-ToggleWide {
+    flex: 1;
+}
+
+.escDshotDirectionDialog-RiskNoticeText {
+    font-size: 1.2em;
+}
+
+.escDshotDirectionDialogInformationNotice {
+    font-size: 1.0em;
+    margin-bottom: 1.5em;
+    margin-top: 1.5em;
+}
+
+#escDshotDirectionDialog-MixerPreview {
+    width: 100%;
+    padding-top: 8px;
+	padding-bottom: 9px;
```

Comment:
Padding is a little off in this file ;-)  Please use right amount of spaces for identation.